examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Actually it is more than that.  <code>triton_op.export</code> is exporting config files as well right? </p><p> </p><p>If yes, I'd say better to write down below what happens after export. We generate model config files so that we can load our models on TIS and send requests successfully.  t </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>clarification added</p>